contracts/starknet/test_contract.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+import os
+import pytest
+
+from starkware.starknet.testing.contract import StarknetContract
+from starkware.starknet.testing.starknet import Starknet
+
+COMPILED_CONTRACTS_DIR = os.path.join("starknet-artifacts", "contracts", "starknet")
```

Comment:
I'm debating whether to take a relative path here or use `__file__` instead. I chose to go with the relative path because it won't break if we move this file into another directory, and it will work in other tests in the same way.

Do you have another suggestion on how to get starknet-artifact's path? Does yarn/hardhat define an environment variable for this?

Reply:
Hardhat retrieves the starknet artifacts path from the param `paths[ "starknetArtifacts"]` in the config file. It currently not set and just using the default which is `starknet-artifacts`. The relative path approach you used seems fine for the python tests though.

Reply:
Cool, Thanks!